tests/micropython/test_cases/test_animation.py

```diff
@@ -0,0 +1,54 @@
+#! ../../../../../../../lv_micropython/ports/unix/build-standard/micropython ../run_testcase.py
```

Comment:
Can you please remove this hard-coded path line?
Or just mention as a simple comment that file should be run with MicroPython executable.
Same in other test files.